src/dmd/target.d

```diff
@@ -312,14 +312,44 @@ extern (C++) struct Target
      */
     void addPredefinedGlobalIdentifiers()
     {
+        alias predef = VersionCondition.addPredefinedGlobalIdent;
```

Comment:
Maybe this should be moved at the previous scope? That will get rid of the duplication.

Reply:
I don't really think that should be a global alias, even if its private to `target.d`

Reply:
I was thinking that it could a private alias inside `struct Target`

Reply:
That won't work because the two uses are in `TargetC` and `TargetCPP` neither of which are subsructs of `Target`

Reply:
Private just means private to the module, and TargetC/CPP could still use it via `Target.predef`.